minutes.md

```diff
@@ -6,9 +6,9 @@ permalink: /minutes/
 
 | Meeting number  | Status | Reference  |
 | --------  | ------------------- | --------------------- |
-| SCM No.30 | Final | <a class="rse" href="/posts/2023-10-24-Minutes-of_SCM-30">2023.10.24</a> |
-| SCM No.29 | Final | <a class="rse" href="/posts/2023-09-30-Minutes-of_SCM-29">2023.09.30</a> |
-| SCM No.28 | Final | <a class="rse" href="/posts/2023-08-31-Minutes-of-SCM-28">2023.08.31</a> |
+| SCM No.30 | Final | <a class="rse" href="/2023/10/24/Minutes-of-SCm-30">2023.10.24</a> |
+| SCM No.29 | Final | <a class="rse" href="/2023/09/30/Minutes-of_SCM-29">2023.09.30</a> |
+| SCM No.28 | Final | <a class="rse" href="/2023/08/31/Minutes-of-SCM-28">2023.08.31</a> |
```

Comment:
Hey sorry - I am not sure I follow the change. Did the filenames change previously? Basically, to my eye it looks like that links will not resolve to correct files.

Reply:
This existing links don't work, and this points to the correct places. You can check by following the links at http://rse-aunz.org/minutes/